### PR TITLE
dnm: Start ovn-kubernetes kind

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -176,6 +176,91 @@ presubmits:
...
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -exc
+        -  |
+          mount
+          sysctl fs.inotify.max_user_watches=1048576
+          sysctl fs.inotify.max_user_instances=512
```
Comment (maiqueb) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:208]: this was it, right  @qinqon ? afaiu podman requires something like this (I run something similar locally ...)

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -176,6 +176,91 @@ presubmits:
...
+          echo "==== describe pod ==="
+          kubectl describe pod -n ovn-kubernetes
+          echo "==== get event ==="
+          kubectl get event -n ovn-kubernetes
+          echo "==== logs ==="
+          for pod in $(kubectl get pod -n ovn-kubernetes -o name); do
+            echo "POD: $pod"
+            kubectl logs -n ovn-kubernetes $pod --all-containers
+          done
+        image: quay.io/kubevirtci/bootstrap-legacy:v20220810-a8f2e6c
```
Comment (maiqueb) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:234]: is this the img we need to pin @qinqon ?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 1 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:208]: also mtu and kind node image

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 2 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:234]: This is exactly the same node image as the SRIOV lane, problem is that we will not be able to bump kind and maybe k8s in the furure, but this applys to SRIOV too I think.

Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:234]: The underlaying problem is the CRIO version at those images and cgroups.

**Pull request closed without merging**